src/main/java/org/reactome/release/dataexport/Main.java

```diff
@@ -12,32 +12,40 @@
  * @author jweiser
  */
 public class Main {
-	@Parameter(names={"--overwrite-config-file", "-c"})
-	private boolean overwriteConfigFile;
+	@Parameter(names={"--generate-config-file", "-g"})
+	private boolean generateConfigFile;
+
+	@Parameter(names={"--config-file-path", "-c"})
+	private String configFilePath;
 
 	/**
 	 * Main method to process configuration file and run the executeStep method of the DataExporterStep class
 	 *
-	 * @param args Command line arguments for the post-release data files export (currently the only argument is,
-	 *     optionally, "--overwrite-config-file" or "-c" to indicate the configuration file should be (re)created
+	 * @param args Command line arguments for the post-release data files export (currently the only arguments are,
+	 * optionally, "--generate-config-file" or "-g" to indicate the configuration file should be (re)created and
+	 * "--config-file-path" or "-c"
 	 * @throws IOException Thrown if unable to create and/or read the configuration file, create output directory
 	 * or write files
 	 */
-	public static void main( String[] args ) throws IOException {
+	public static void main(String[] args) throws IOException {
 		Main main = new Main();
 		JCommander.newBuilder()
-			.addObject(new Main())
+			.addObject(main)
 			.build()
 			.parse(args);
 
 		main.run();
 	}
 
 	private void run() throws IOException {
-		ConfigurationManager configurationManager = new ConfigurationManager();
-		configurationManager.createConfigurationFile(overwriteConfigFile);
+		ConfigurationManager configurationManager = new ConfigurationManager(configFilePath);
+		configurationManager.createConfigurationFile(!configFilePathExists() && generateConfigFile);
```

Comment:
`createConfigurationFile` does not seem like an accurate name if it could read an existing file...

Reply:
Okay, I can rename this.

Reply:
Okay, done